vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/get_retirement_entrypoint.rb

```diff
@@ -7,14 +10,5 @@
   $evm.log("info", "automate_retirement_entrypoint setting default entry_point to: #{entry_point} ")
 end
 
-parts = entry_point.split('/')
-parts.shift  if entry_point[0, 1]  == '/'
-retirement_instance    = parts.pop
-retirement_class       = parts.pop
-retirement_ns          = parts.join('/')
-
-$evm.root['retirement_ns']       = retirement_ns
-$evm.root['retirement_class']    = retirement_class
-$evm.root['retirement_instance'] = retirement_instance
-
-$evm.log("info", "entry_point: ns: #{retirement_ns} class: #{retirement_class} instance: #{retirement_instance} ")
+$evm.root['retirement_entry_point']	  = entry_point
+$evm.log("info", "entry_point: #{entry_point} ")
```

Comment:
Clean up the extra spaces before the equal sign.  Also, this last log line is can be removed since we are logging the entry_point above.